nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -58,6 +58,26 @@ let
 
   snapshotNames = [ "frequent" "hourly" "daily" "weekly" "monthly" ];
 
+  importLib = {pool, cfgZfs}: ''
+    zpool_cmd="${packages.zfsUser}/sbin/zpool"
+    awk_cmd="${pkgs.gawk}/bin/awk"
```

Comment:
Maybe use the busybox awk instead

Reply:
Now there's an idea.

Well, it worked, but it didn't affect the closure size. I'm starting to wonder if there's a spurious dependency from the scripting, somehow.

Reply:
...false. It seems I'm getting too sleepy for this, as I forgot to `git commit`.

Swapping to busybox awk fixes the closure size... though it doesn't tell me why that was a problem to begin with.

Reply:
...false, again. I'll get back to this when I'm more awake, but it happens independently of the awk selection.